### PR TITLE
add badges endpoint

### PR DESCRIPTION
**Pull request by krokrob**
this add a `api/:username/badges` endpoint to get all badges as a JSON

### CODE REVIEW

**Comment by krokrob** [comment]
@ssaunier please review

File: views/index.erb [hunk 1]
```diff
@@ -1,4 +1,5 @@
 <form action="/user" method="post">
   <input type="text" name="nickname">
   <input type="submit">
-</form>
\ No newline at end of file
+</form>
+<%= ENV['CODECADEMY_USERNAME'] %>
```
Comment (ssaunier) [inline views/index.erb]: Tu veux laisser ça?

Comment (krokrob) [reply views/index.erb]: non

**Comment by ssaunier** [comment]
I redeployed it 👍

**Pull request merged**